Cesium.CodeGen.Tests/verified/CodeGenMethodTests.StructSubscription.verified.txt

```diff
@@ -0,0 +1,21 @@
+﻿System.Int32 <Module>::main()
+  Locals:
+    foo V_0
+  IL_0000: ldloca.s V_0
+  IL_0002: conv.u
+  IL_0003: ldc.i4.0
+  IL_0004: stfld foo/<SyntheticBuffer>fixedArr foo::fixedArr
+  IL_0009: ldloca.s V_0
+  IL_000b: conv.u
+  IL_000c: ldfld foo/<SyntheticBuffer>fixedArr foo::fixedArr
+  IL_0011: ret
```

Comment:
Now you can see that this doesn't include `[3]` anywhere. This is a problem.

Reply:
@te9c do you need some help with finishing this?

Reply:
@kant2002 yes, i would be grateful, just don't understand how we can return array element from field?

Reply:
@te9c I think after https://github.com/ForNeVeR/Cesium/pull/273 lands, you can continue poking at this PR after you rebase on main.

Reply:
@te9c can you rebase/merge on main and try move forward?